.circleci/config.yml

```diff
@@ -150,8 +150,8 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Install protobuf and git
-          command: pacman -Sy --noconfirm protobuf git && protoc --version
+          name: Install protobuf and git and dependencies
+          command: pacman -Sy --noconfirm protobuf git gcc-libs && protoc --version
```

Comment:
This is the fix for https://app.circleci.com/pipelines/github/influxdata/pbjson/274/workflows/b620d772-632d-4dd8-bc83-cd9a0e01a30b/jobs/1432

It appears the new version of protoc needs its libraries updated as well